egs/iwslt18/asr1/run.sh

```diff
@@ -43,6 +48,13 @@ case=lc.rm
 st_ted=/export/b08/inaguma/IWSLT
 # st_ted=/n/rd11/corpora_8/iwslt18
 
+# bpemode (unigram or bpe)
+nbpe=1000
+bpemode=bpe
+# NOTE: nbpe=69 means character-level ASR (case=lc.rm)
```

Comment:
I like this way, but another possibility is to get this number automatically when we set `nbpe=0`. But this may make `run.sh` complicated, and the current way is good for me.

Reply:
nbpe=0 causes an error in spm_train, which requires the second run. I don't prefer it.